modules/round/src/main/GameProxyRepo.scala

```diff
@@ -58,4 +61,29 @@ final class GameProxyRepo(
         }
       }
     }
+
+  def correspondenceGamesToPlayOf(
+      users: List[lila.user.User]
+  ): Fu[List[CorrespondenceOpponents]] =
+    (users map { user =>
+      gameRepo urgentPovsUnsorted user flatMap {
+        _.map { pov =>
+          gameIfPresent(pov.gameId) dmap { _.fold(pov)(pov.withGame) }
+        }.sequenceFu map { povs =>
```

Comment:
`sequenceFu` runs all these queries in parallel (as its name doesn't indicate). Safer to use `lila.common.Future.applySequentially` or `lila.common.Future.fold`. There are examples of use in lila.